tests/unit/fem/test_transfer.cpp

```diff
@@ -230,6 +508,7 @@ TEST_CASE("partransfer", "[Parallel]")
                                << "  Fine order:   " << fineOrder << "\n"
                                << "  Geometric:    " << geometric << "\n";
                   }
+                  coeff_order = std::min(1,order);
```

Comment:
Did you mean `max`? This will always be 1.

Reply:
Yes this is supposed to be 1 always. I should just put `coeff_order =  1 `